tests/model_serving/model_runtime/vllm/utils.py

```diff
@@ -51,13 +51,16 @@ def fetch_openai_response(  # type: ignore
     model_name: str,
     chat_query=CHAT_QUERY,
     completion_query=COMPLETION_QUERY,
+    tool_calling: Optional[dict[Any, Any]] = None,
```

Comment:
please use  union (X | None)  https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
Aside from personal preference, what benefits do you see? I’ve tried using both options, but it didn’t seem to make a difference in the test itself. If you’ve noticed any impact with suggested changes, I’d like to know.

Reply:
https://docs.python.org/3/library/stdtypes.html#union-type

"The union type expression enables cleaner type hinting syntax "